### ISSUE TITLE
Hard to follow documentation

### ISSUE THREAD

**Issue by brittonwalker** [FunTechInc/mekuri#75]
I love the ability to transition in-out simultaneously here but can't seem to recreate it myself. I assume the code in here is the same thing that follows the demo. Not sure what or if I'm doing anything wrong but I would love to understand it.

**Comment by takuma-hmng8** [comment]
@brittonwalker 
Sorry, I am currently dedicated to another project and have not had time to maintain this library.
If you want to reproduce an animation like the one on the demo page, please refer to the actual project file.
/app has it all.
